### PR TITLE
feat(tangle-dapp): Add Restake Withdraw Flow

### PR DESCRIPTION
**Pull request by AtelyPham**
## Summary of changes

_Provide a detailed description of proposed changes._

- [x] Implement scheduled withdrawals for restaking  
- [x] Retrieve and display withdrawable assets for restaking  
- [x] Create a table for scheduled withdrawal requests  
- [x] Add an action to cancel withdrawal requests  
- [x] Add an action to execute withdrawal requests

### Proposed area of change

_Put an `x` in the boxes that apply._

- [ ] `apps/bridge-dapp`
- [ ] `apps/hubble-stats`
- [ ] `apps/stats-dapp`
- [x] `apps/tangle-dapp`
- [ ] `apps/testnet-leaderboard`
- [ ] `apps/faucet`
- [ ] `apps/zk-explorer`
- [ ] `libs/webb-ui-components`

### Reference issue to close (if applicable)

_Specify any issues that can be closed from these changes (e.g. `Closes #233`)._

- Closes https://github.com/webb-tools/webb-dapp/issues/2408

### Screen Recording

_If possible provide a screen recording of proposed change._

https://github.com/user-attachments/assets/456069d2-0bc6-4fd1-abd8-0ce81b7ddccb

### CODE REVIEW

**Comment by drewstone** [comment]
Does this handle EVM precompile interactions too if the logged in user is from the EVM?

File: apps/tangle-dapp/app/restake/withdraw/TxInfo.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+import { useMemo } from 'react';
+
+import useRestakeConsts from '../../../data/restake/useRestakeConsts';
+
+const TxInfo = () => {
+  const { leaveDelegatorsDelay } = useRestakeConsts();
+
+  const items = useMemo<FeeItem[]>(
+    () => [
+      // TOOD: Add fee value
```
Comment (drewstone) [inline apps/tangle-dapp/app/restake/withdraw/TxInfo.tsx:12]:
```suggestion
      // TODO: Add fee value
```

Comment (drewstone) [reply apps/tangle-dapp/app/restake/withdraw/TxInfo.tsx:12]: What fee value is this?

**Review by drewstone — APPROVED** [review]
Small nit, and if you can integrate EVM precompile alongside this that would be ideal.

**Comment by AtelyPham** [comment]
> Small nit, and if you can integrate EVM precompile alongside this that would be ideal.

I'm currently integrating the precompile into the restaking, which should be completed today. 

Regarding the fee value, it refers to the transaction fee. I've noted this in the design but haven't implemented it yet, so I've added a `TODO` in the code.

**Pull request merged**